item.rb

```diff
@@ -0,0 +1,26 @@
+require 'date'
+
+class Item
```

Comment:
- [ ] As required for the project, all public methods in all classes to have unit tests, kindly ensure that you add tests to the `Item` class and perform tests to the public method(s) inside it.